test/test_modules_taken_before_others.py

```diff
@@ -0,0 +1,120 @@
+'''
+    test_modules_before_others_ui.py tests the page views for
```

Comment:
docstring file name is different from the actual file name. 
The docstring name is "test_modules_before_others_ui.py"
while the real name is "test_modules_taken_before_others.py"